### ISSUE TITLE
Build error: configure: error: libtracefs is required but was not found

### ISSUE THREAD

**Issue by vrossum** [fenrus75/powertop#124]
I get a build error on Ubuntu22.04
configure: error: libtracefs is required but was not found

> apt install -y libtracefs1
returns already installed

**Comment by fenrus75** [comment]
you're going to need to install the -dev version of that package, not just
the library

**Comment by vrossum** [comment]
You are right. But it is already installed as well.

apt install -y libtracefs-dev
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
libtracefs-dev is already the newest version (1.2.5-1).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

**Comment by yeongjoshua** [comment]
This fix for me

```
git clone https://git.kernel.org/pub/scm/libs/libtrace/libtraceevent.git/
cd libtraceevent; make; sudo make install; cd ..;

git clone https://git.kernel.org/pub/scm/libs/libtrace/libtracefs.git/
cd libtracefs; make; sudo make install; cd ..;
```

**Comment by ivan-koryshkin** [comment]
Same problem

`configure: error: libtracefs is required but was not found`

I have installed `libtracefs-dev` and `libtracefs1` in my system

**Comment by stephematician** [comment]
I had to install `libtracefs1`, `libtracefs-dev`, `libtraceevent1` and `libtraceevent-dev`. The former two were insufficient on their own.

**Comment by xmixahlx** [comment]
FYI resolved with installing libtracefs-dev and libtraceevent-dev on Debian Unstable.

**Comment by Hoernchen** [comment]
Same issue on ubuntu 24.04, fixed by installing libtraceevent-dev.

**Comment by ValeZAA** [comment]
Yep, Debian testing, no issue with tracefs_event_file_exists, just install libtraceevent-dev

This is not mentioned in Readme though under

Example packages to install in Ubuntu*:
